pkg/pdapi/pdapi.go

```diff
@@ -698,3 +706,23 @@ func getLeaderEvictSchedulerInfo(storeID uint64) *schedulerInfo {
 func getLeaderEvictSchedulerStr(storeID uint64) string {
 	return fmt.Sprintf("%s-%d", "evict-leader-scheduler", storeID)
 }
+
+// TiKVNotBootstrappedError represents that TiKV cluster is not bootstrapped yet
+type TiKVNotBootstrappedError struct {
```

Comment:
Should this and related types go into `pkg/manager/member/utils.go`?

Reply:
I think this is the error that belongs to `pd api`, maybe suitable to place it here.